utils/results.go

```diff
@@ -81,41 +81,12 @@ func (r *Results) IsIssuesFound() bool {
 
 // Counts the total number of unique findings in the provided results.
 // A unique SCA finding is identified by a unique pair of vulnerability's/violation's issueId and component id or by a result returned from one of JAS scans.
-func (r *Results) CountScanResultsFindings() (total int) {
-	return formats.SummaryResults{Scans: r.getScanSummaryByTargets()}.GetTotalIssueCount()
-}
-func (r *Results) GetSummary() (summary formats.SummaryResults) {
-	if len(r.ScaResults) <= 1 {
-		summary.Scans = r.getScanSummaryByTargets()
-		return
-	}
-	for _, scaScan := range r.ScaResults {
-		summary.Scans = append(summary.Scans, r.getScanSummaryByTargets(scaScan.Target)...)
-	}
-	return
-}
-
-// Returns a summary for the provided targets. If no targets are provided, a summary for all targets is returned.
-func (r *Results) getScanSummaryByTargets(targets ...string) (summaries []formats.ScanSummaryResult) {
-	if len(targets) == 0 {
-		// No filter, one scan summary for all targets
-		summaries = append(summaries, getScanSummary(r.ExtendedScanResults, r.ScaResults...))
-		return
+func (r *Results) CountScanResultsFindings(includeVulnerabilities, includeViolations bool) (total int) {
```

Comment:
I see we always pass (true,true) to this function - and from what I see - we never return the vuln - only violations

Reply:
For the first note - I can remove the inputs to the func.
About the second note - we return the violations count if exists (> 0) or the vulnerabilities count if no violations
![image](https://github.com/user-attachments/assets/1482ac4a-0a37-4bad-9a55-0f7e13c4d3d8)

Reply:
ok - so if there are violations - we'll never show the vulnerabilities